src/distPlus/symbolic/SymbolicDist.re

```diff
@@ -47,12 +47,31 @@ type dist = [
   | `Cauchy(cauchy)
   | `Triangular(triangular)
   | `ContinuousShape(continuousShape)
-  | `Float(float)
+  | `Float(float) // Dirac delta at x. Practically useful only in the context of multimodals.
 ];
 
-type pointwiseAdd = array((dist, float));
+type integral = float;
+type cutoffX = float;
+type operation = [
+    | `AddOperation
+    | `SubtractOperation
+    | `MultiplyOperation
+    | `DivideOperation
+    | `ExponentiateOperation
+];
 
-type bigDist = [ | `Simple(dist) | `PointwiseCombination(pointwiseAdd)];
+type distTree = [
+    | `Simple(dist)
```

Comment:
"Simple" was chosen pretty quickly. Maybe something like "raw" or "singleDistribution", or you could come up with some better name?

Reply:
Yeah I've gone back and forth a few times and reverted back to Simple for now. "Symbolic", "Raw", "Atomic", "Simple", "Single", none of them are quite the *mot juste* here :shrug:

Reply:
That's fine, I don't have very strong feelings here.